src/main/scala/com/gu/anghammarad/EmailService.scala

```diff
@@ -0,0 +1,46 @@
+package com.gu.anghammarad
+
+import com.amazonaws.auth.{AWSCredentialsProviderChain, EnvironmentVariableCredentialsProvider}
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.simpleemail.model.{Body, Content, Destination, SendEmailRequest, SendEmailResult, Message => AwsMessage}
+import com.amazonaws.services.simpleemail.{AmazonSimpleEmailService, AmazonSimpleEmailServiceClientBuilder}
+import scala.util.Try
+
+object EmailService {
+
+  private val provider = new AWSCredentialsProviderChain(
```

Comment:
As you said elsewhere, at some point we should pull this out so it can be used for other functionality as well.